composer.json

```diff
@@ -10,7 +10,8 @@
     "cakephp/plugin-installer": "*"
   },
   "require-dev": {
-    "phpunit/phpunit": "4.1.*"
+    "phpunit/phpunit": "4.1.*",
+    "loadsys/loadsys_codesniffer": "~3.0"
```

Comment:
Sure, a reasonable action.

I think those notes were put in place before we had a `composer.json.template` version of the file though.